workflow/rules/consensus_peak_analysis.smk

```diff
@@ -85,7 +92,8 @@ rule create_consensus_igv:
     log:
         "logs/igv/consensus/merged_library.{antibody}.consensus_{peak}-peaks.igv.log"
     shell:
-        "find {input} -type f -name '*.consensus_{wildcards.peak}-peaks.boolean.bed' -exec echo -e 'results/IGV/consensus/{wildcards.antibody}/\"{{}}\"\t0,0,0' \; > {output} 2> {log}"
+        "find {input} -type f -name '*.consensus_{wildcards.peak}-peaks.boolean.bed' -exec "
```

Comment:
It is meant to write each file from input (entire path) together with `\t0,0,0` line by line into the output file. Alternatively we could construct this in a for-loop. But I would not recommend this, because `find` is much more efficient.

Reply:
I still think the way the nextflow pipeline does it is a bit hacky, but it's probably the only way this is feasible with their channel system. I would rather go for a Snakemakey solution, where all the requires input files are generated using an input function, so you then have them available via `input`. You can then go and simply do string manipulation on the full list of file names in `input` via a `lambda` expression in the `params:`, that also appends the `\t0,0,0`, and then just write out the resulting list to the `output` file. Am I making sense?